tests/utils/dockercli/volumelifecycle.go

```diff
@@ -53,8 +65,35 @@ func KillDocker(ip string) ([]byte, error) {
 	return out, err
 }
 
+// KillVDVSPlugin - kill docker daemon. It is restarted automatically
+func KillVDVSPlugin(ip string) ([]byte, error) {
+	log.Printf("Killing vDVS plugin on VM [%s]\n", ip)
+	out, err := ssh.InvokeCommand(ip, dockercli.GetVDVSPlugin)
+	if err != nil {
+		return out, err
+	}
+
+	pluginID := strings.Fields(misc.FormatOutput(out))[0]
+	out, err = ssh.InvokeCommand(ip, dockercli.KillVDVSPlugin+pluginID)
+
+	time.Sleep(2 * time.Second)
```

Comment:
we should extract such wait time into a helper so it will be easier to control in future.